packages/backend/src/apps/formsg/triggers/new-submission/get-data-out-metadata.ts

```diff
@@ -65,6 +102,8 @@ async function getDataOutMetadata(
       answer: buildAnswerMetadatum(fieldData),
```

Comment:
I actually have a bug here where I don't handle lack of `answer` correctly <.<

Reply:
wait when will there be no answer though?